Translation/en/RedirectedResources/assets/abdata/h/list/71/personality_voice_c15/zz_machineTranslation.txt

```diff
@@ -1,2 +1,2 @@
-（ねっとり絡みつくあたしの舌……気持ちいいだろ？）=(My tongue is soggy and entwined ... it feels good, right?)
+（ねっとり絡みつくあたしの舌……気持ちいいだろ？）=
```

Comment:
@ManlyMarco I know this is merged already, but what happened here?

Reply:
No idea, it must have happened at some point when cleaning the MTL. Thankfully this seems to be the only case, I'll fix it in a bit.